ci/build-linux.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [ "x${TRAVIS_OS_NAME}" != "xlinux" ]; then
+  echo "Not a Linux-based build, exit successfully."
+  exit 0
+fi
+
+IMAGE="cached-${DISTRO?}-${DISTRO_VERSION?}"
```

Comment:
`declare -r IMAGE=...`

Reply:
Good point, but `readonly` is kind of more readable?

Reply:
I forget which of `readonly` or `declare -r` is more standard or more widely used; https://google.github.io/styleguide/shell.xml?showone=Read-only_Variables#Read-only_Variables suggests using either one of them, so `readonly` SGTM.